contrib/mesos/pkg/scheduler/podtask/registry.go

```diff
@@ -186,7 +186,7 @@ func (k *inMemoryRegistry) UpdateStatus(status *mesos.TaskStatus) (*T, StateType
 		k.handleTaskRunning(task, state, status)
 	case mesos.TaskState_TASK_FINISHED:
 		k.handleTaskFinished(task, state, status)
-	case mesos.TaskState_TASK_FAILED:
+	case mesos.TaskState_TASK_FAILED, mesos.TaskState_TASK_ERROR:
```

Comment:
there's a separate PR for this work, yes? was this needed to workaround a testing blocker?